tests/ci/docker_images/linux-x86/ubuntu-22.04_base/Dockerfile

```diff
@@ -47,9 +51,12 @@ RUN set -ex && \
     unzip awscliv2.zip && \
     ./aws/install --bin-dir /usr/bin && \
     rm -rf awscliv2.zip aws/ && \
-    # Download a copy of LLVM's libcxx which is required for building and running with Memory Sanitizer
     mkdir -p ${DEPENDENCIES_DIR} && \
     cd ${DEPENDENCIES_DIR} && \
+    # Extract and install Boost, dependency needed for mySQL.
+    wget https://boostorg.jfrog.io/artifactory/main/release/1.77.0/source/${BOOST_PACKAGE_NAME}.tar.bz2 && \
+    tar xfj ${BOOST_PACKAGE_NAME}.tar.bz2 && mv ./${BOOST_PACKAGE_NAME} ./boost && \
```

Comment:
* Since both the URL and variable `BOOST_PACKAGE_NAME` imply version number "1.77.0", I'm not sure that the variable provides much value. For visibility, perhaps the variable should contain the whole URL.
* Do you specifically need version 1.77.0? My Ubuntu (22.04) host has a package for Boost "1.74.0":
```
❯ apt-cache show libboost-all-dev | head -n 3
Package: libboost-all-dev
Architecture: amd64
Version: 1.74.0.3ubuntu7
```

Reply:
* Sure, I took this from mySQL, but I think we can clean this up a bit.
* Yes, the mySQL build requires at least 1.77.0 which is why I had to install it from source. I should have clarified that in the comments, I'll add that.
```
CMake Warning at cmake/boost.cmake:305 (MESSAGE):
  Boost minor version found is 74 we need 77
```